test/rest_request_template/schema_evolution_nullable_values_after_smt.json

```diff
@@ -15,6 +15,7 @@
     "snowflake.schema.name": "SNOWFLAKE_SCHEMA",
     "snowflake.role.name": "SNOWFLAKE_ROLE",
     "snowflake.ingestion.method": "SNOWPIPE_STREAMING",
+    "snowflake.streaming.closeChannelsInParallel.enabled": true,
```

Comment:
can we add a test with false to make sure rollback is working like current behavior?

Reply:
I'll try to test this in a unit test.
Each e2e test is run about 45 times, so I'd like to avoid writing e2e's for functionality that can be tested within a quicker set of tests.

Reply:
Added [CloseTopicPartitionChannelIT](https://github.com/snowflakedb/snowflake-kafka-connector/pull/841/files#diff-24f62287e02cb6967f8a0c0792c4bab2ef5fcec5460003c41b951daf399dc6f1R17)  that ensures all channels are closed no matter what the config value is.